px-kpi-list.html

```diff
@@ -138,7 +140,19 @@
        */
       footer: {
         type: String
-      }
+      },
+	 /**
+      * This will check if label or statusIcon or statusLabel is present or not
+	  * if not then don't show header else show header 
+      *
+      * @property size
+      */
+	_checkIsHeaderDataPresent() {
+		if(this.label != undefined || this.statusIcon != undefined || this.statusLabel != undefined)
```

Comment:
Violates the `eqeqeq` lint rule. Also, this could probably be condensed to:
```javascript
_checkIsHeaderDataPresent() {
  return !!this.label || !!this.statusIcon || !!this.statusLabel;
}
```